source/format.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let format = (input, columns) => {
+	if (columns < 1) {
+		return `Amount of coloms could't be ${columns}`
+	}
+	if (columns > input.length) {
+		return `Amount of coloms could't be bigger than amount of numbers (${columns}>${input.length})`
+	}
+	let output = Array(Math.ceil(input.length / columns)).fill("");
```

Comment:
Не очень понимаю смысл в этом заполненном массиве, кажется, что можно обойтись пустым массивом
```suggestion
	const rows = [];
```

Reply:
Разве с алгоритмической точки зрения, не быстрее создать массив нужной итоговой длинны и обращаться по индексам чем добавлять в конец массива значения увеличивая его?

Reply:
Да, твой тейк понятен. Можем оставить так, я в угоду читаемости готов пожертвовать немного скоростью работы